backend/ddueruemweb/settings.py

```diff
@@ -69,6 +69,7 @@
         ,
         'APP_DIRS': True,
         'OPTIONS': {
+            'debug': True,
```

Comment:
debug in "production" ? Might be discussible.

Reply:
add settings_dev.py

Reply:
So we probably need a script in the production and dev pipeline to set the DJANGO_SETTINGS_MODULE
as described in here
https://stackoverflow.com/a/10664412/12263454

Reply:
I added a new variable TEMPLATE_DEBUG into the env files. The settings.py file is also modified to use this variable